tech-ally/src/app/models/repo.ts

```diff
@@ -8,24 +9,37 @@ export class Repo {
     iconLink: string = "";
     language: string;
     description: string;
+    latestRelease: string;
+    releasePublished: string;
+    releaseLink: string;
+    commits!: Commit[];
 
-    constructor(name: string, issues: number, pullRequests: number, language: string, description: string) {
+    constructor(name: string, issues: number, pullRequests: number, language: string, description: string, 
```

Comment:
nit: this constructor is getting pretty big 😅 shouldn't we pass an object/interface? 🤔